### PR TITLE
Rename `namespace` to `userCodeNamespace` in various places

### PR DESCRIPTION
**Pull request by JamesHazelcast**
Updated as part of a wider effort to bring "UCD namespace" naming into consistency under "User Code Namespace".

Mono PR: https://github.com/hazelcast/hazelcast-mono/pull/272

### CODE REVIEW

File: protocol-definitions/DynamicConfig.yaml [hunk 1]
```diff
@@ -1405,10 +1405,10 @@ methods:
             The WAN batch publisher configurations.
     response: {}
   - id: 19
-    name: addNamespaceConfig
+    name: addUserCodeNamespaceConfig
     since: 2.7
     doc: |
-      Adds a namespace configuration.
+      Adds a user code namespace configuration.
```
Comment (JackPGreen) [inline protocol-definitions/DynamicConfig.yaml:1411]: I think the doc on the parameters needs to change as well

File: protocol-definitions/DynamicConfig.yaml [hunk 2]
```diff
@@ -1427,7 +1427,7 @@ methods:
             List of resource definitions.
     response: {}
   - id: 20
-    name: removeNamespaceConfig
+    name: removeUserCodeNamespaceConfig
```
Comment (JackPGreen) [inline protocol-definitions/DynamicConfig.yaml:1430]: I think the doc on the parameters needs to change as well

**Review by JackPGreen — CHANGES_REQUESTED** [review]

File: protocol-definitions/DynamicConfig.yaml (hunk 1 above)
Comment (JamesHazelcast) [reply protocol-definitions/DynamicConfig.yaml:1411]: The doc is already updated, as shown in the diff above.

File: protocol-definitions/DynamicConfig.yaml (hunk 2 above)
Comment (JamesHazelcast) [reply protocol-definitions/DynamicConfig.yaml:1430]: Fixed this missing doc in https://github.com/hazelcast/hazelcast-client-protocol/pull/495/commits/5b1ae0e786997c22e4d1d837baa148dd03daef7e

**Review by JackPGreen — APPROVED** [review]

**Review by vbekiaris — APPROVED** [review]

**Pull request merged**